codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 60%
```

Comment:
Perhaps make sense to use `auto` for target? (auto compares coverage to the previous base commit)
So that we do not need to keep on updating the target?

Perhaps also add:
`require_ci_to_pass: true`
so that it is only done after CI passes?

Reply:
Regarding `require_ci_to_pass` This will increase the total execution time. If there is no need to reduce the CI load I would not set it.

Regarding target: The per patch settings are auto by default. I'm not so sure why the bot was failing with the default settings for PRs that  increased the coverage

Reply:
Interesting thing about the `auto`. Let's leave it like that then.